assets/src/common/helpers/index.js

```diff
@@ -336,22 +297,6 @@ export const mediaLibraryHasTwoNotices = function() {
 	return Boolean( this.secondary.get( FILE_TYPE_ERROR_VIEW ) ) && Boolean( this.secondary.get( FILE_SIZE_ERROR_VIEW ) );
 };
 
-/**
- * Gets the number of megabytes per second for the video.
- *
- * @param {Object} media The media object of the video.
- * @return {?number} Number of megabytes per second, or null if media details unavailable.
- */
-export const getVideoBytesPerSecond = ( media ) => {
-	if ( has( media, [ 'media_details', 'filesize' ] ) && has( media, [ 'media_details', 'length' ] ) ) {
-		return media.media_details.filesize / media.media_details.length;
-	} else if ( has( media, [ 'attributes', 'filesizeInBytes' ] ) && has( media, [ 'attributes', 'fileLength' ] ) ) {
-		return media.attributes.filesizeInBytes / getSecondsFromTime( media.attributes.fileLength );
-	}
-
-	return null;
-};
-
 /**
  * Gets whether the video file size is over a certain amount of bytes per second.
  *
```

Comment:
At least the following functions can also be removed:

`isVideoSizeExcessive`
`getSecondsFromTime`
`getContentLengthFromUrl`
`getBackgroundColorWithOpacity`
`getRgbaFromHex`
`mediaLibraryHasTwoNotices` (and logic around it)

Also some constants:

`MEGABYTE_IN_BYTES`
`VIDEO_ALLOWED_MEGABYTES_PER_SECOND`
`FILE_SIZE_ERROR_VIEW`